src/apis/avm/api.ts

```diff
@@ -571,7 +571,7 @@ export class AVMAPI extends JRPCAPI {
   getAllBalances = async (address:string):Promise<Array<object>> => {
     if (typeof this.parseAddress(address) === 'undefined') {
       /* istanbul ignore next */
-      throw new Error(`Error - AVMAPI.getAllBalances: Invalid address format ${address}`);
+      throw new Error(`Error - AVMAPI.getAllBalances: Invalid address format`);
```

Comment:
For style should we replace the back-ticks with `" "` as I am under the impression back-ticks should be reserved for interpolation.

Reply:
Resolved